components/AudioPlayer.module.css

```
@@ -0,0 +1,198 @@
+.audio_player_section_container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  padding: 10%;
+}
+
+.image_wrapper {
+  position: relative;
+  top: 30px;
+  width: 180px;
+  height: 180px;
+  margin: 0 auto;
+  box-shadow: 0 0 20px black;
+}
+
+.episode_info_section{
+  position: relative;
+  top: 50px;
+  display: block;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.episode_title {
+  text-align: center;
+}
+
+.episode_details{
+  width: 100%;
+  height: 150px;
+  overflow: scroll;
+  text-align: start;
+}
+
+
+.audioPlayer {
+    --primary: grey;
+    --secondary: white;
+    position: absolute;
+    bottom: 10%;
+    width: 80%;
+    text-align: center;
+  }
+
+  .audioPlayerPlaybutton{
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+    margin: 20px;
+  }
+  
+  .forwardBackward {
```

```suggestion
  .controlThirty {
```
**Suggestion:** it was used for the forward button as well.  Naming `forwardBackward` might confuse so I suggest that name more general name like this.